toolkit/tools/pkg/imagecustomizerlib/savedconfigs.go

```diff
@@ -29,11 +29,17 @@ import (
 // re-apply them.
 
 type LiveOSSavedConfigs struct {
-	KernelCommandLine imagecustomizerapi.KernelCommandLine `yaml:"kernelCommandLine"`
+	KdumpBootFiles    *imagecustomizerapi.KdumpBootFilesType `yaml:"kdumpBootFiles"`
+	KernelCommandLine imagecustomizerapi.KernelCommandLine   `yaml:"kernelCommandLine"`
 }
 
 func (i *LiveOSSavedConfigs) IsValid() error {
-	err := i.KernelCommandLine.IsValid()
+	err := i.KdumpBootFiles.IsValid()
```

Comment:
Null check the pointer?

Though I would just remove the pointer as per my other comment.